### ISSUE TITLE
Build error with examples-pse 1.8.0 in file Tutorials/Advanced/ParamEst/DMF_flash_unit_Model_with_NRTL_solution.ipynb

### ISSUE THREAD

**Issue by lbianchi-lbl** [IDAES/examples-pse#23]
```text
2021-01-13 17:17:46,313 ERROR - [Worker 0] Notebook execution failed: execution error for '/tmp/tmpdz3kfiy4/DMF_flash_unit_Model_with_NRTL_solution.ipynb': An error occurred while executing the following cell:
------------------
from idaes.generic_models.unit_models import Flash
m.fs.flash = Flash(default={"property_package": m.fs.properties})
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-6a6bc63bd76a> in <module>
      1 from idaes.generic_models.unit_models import Flash
----> 2 m.fs.flash = Flash(default={"property_package": m.fs.properties})

~/opt/conda/envs/examples-rel/lib/python3.7/site-packages/pyomo/core/base/block.py in __getattr__(self, val)
    522         # throw the "normal" AttributeError
    523         raise AttributeError("'%s' object has no attribute '%s'"
--> 524                              % (self.__class__.__name__, val))
    525 
    526     def __setattr__(self, name, val):

AttributeError: '_ScalarFlowsheetBlock' object has no attribute 'properties'
AttributeError: '_ScalarFlowsheetBlock' object has no attribute 'properties'

2021-01-13 17:17:46,313 ERROR - [Worker 0] Execution failed: generating partial output for '/home/lbianchi/idaes/examples-pse-rel/src/Tutorials/Advanced/ParamEst/DMF_flash_unit_Model_with_NRTL_solution_testing.ipynb'
```

**Comment by ksbeattie** [comment]
Need to look at why this is not caught by our integration tests.

**Comment by lbianchi-lbl** [comment]
This was fixed in #32.
